command/results_gotest.go

```diff
@@ -117,16 +116,13 @@ func (c *goTestResults) Execute(ctx context.Context,
 // AllOutputFiles creates a list of all test output files that will be parsed, by expanding
 // all of the file patterns specified to the command.
 func (c *goTestResults) allOutputFiles() ([]string, error) {
-
-	outputFiles := []string{}
-
-	// walk through all specified file patterns
-	for _, pattern := range c.Files {
-		matches, err := filepath.Glob(pattern)
-		if err != nil {
-			return nil, errors.Wrap(err, "error expanding file patterns")
-		}
-		outputFiles = append(outputFiles, matches...)
+	dir, err := os.Getwd()
+	if err != nil {
+		return nil, errors.Wrap(err, "error retrieving working directory path")
+	}
+	outputFiles, err := getFilePaths(dir, c.Files)
```

Comment:
`getFilePaths` uses `filepath.Glob`, but this ticket is about gitignore-style globbing.

Reply:
Sorry can you explain what you mean? getFilePaths has been modified to use util.BuildFileList rather than filepath.Glob, which does support gitignore-style globbing, unless I misunderstand the ticket/your comment.

Reply:
use this: https://github.com/evergreen-ci/evergreen/blob/0e7cfad84bde60c153e96a18c5af144d20ec3f01/util/file.go#L71

Reply:
Right, getFilePath does call that.

Reply:
Oh, I totally missed that you changed `getFilePaths` to call `BuildFileList`, sorry. Are you confident about `getFilePaths`'s test coverage? LGTM if yes.